algosdk/atomic_transaction_composer.py

```diff
@@ -754,19 +784,56 @@ def simulate(
                     decode_error=result.decode_error,
                     tx_info=result.tx_info,
                     method=result.method,
-                    missing_signature=sim_txn.get("missing-signature", False),
                 )
             )
 
         return SimulateAtomicTransactionResponse(
             version=simulation_result.get("version", 0),
-            would_succeed=simulation_result.get("would-succeed", False),
             failure_message=txn_group.get("failure-message", ""),
             failed_at=txn_group.get("failed-at"),
             simulate_response=simulation_result,
             tx_ids=self.tx_ids,
             results=sim_results,
+            eval_overrides=SimulateEvalOverrides.from_simulation_result(
+                simulation_result
+            ),
+        )
+
+    def simulate_with_request(
```

Comment:
Can we combine this with the existing `simulate` method? I don't see the point in having both

The added logic in this function to accommodate signed vs unsigned txns is interesting, but I'm not convinced it's necessary. #420 added `EmptySigner`, which can be used in the ATC if the caller really doesn't want to simulate with signatures

Reply:
ok! that sounds great to me, for I was not aware of `EmptySigner` existence, but that def helps me to transform from `txn` to `stxn` easily.